src/test/espn_test.py

```diff
@@ -0,0 +1,31 @@
+import unittest
+from scrape_up.espn import ESPN
+
+class ESPNScraperTest(unittest.TestCase):
+    """
+    ESPN class test.\n
+    | Methods              | Details                                              |
+    | -------------------  | ---------------------------------------------------- |
+    | `.get_scoreboard()`  | Tests the get_scoreboard() method of the ESPN class  |
+    | `.get_tournaments()` | Tests the get_tournaments() method of the ESPN class |
+    | `.get_teams()`       | Tests the get_teams() method of the ESPN class       |
+    """
+
+    def test_get_scoreboard(self):
+        espn = ESPN()
+        date = "20230721"  # Replace with a valid date
+        scoreboard = espn.get_scoreboard(date)
+        self.assertIsInstance(scoreboard, list)
```

Comment:
Not only check the response instance type, but also check whether it has all the keys or not. 
For example, in the `codechef_test.py` script, we have also checked the key value of the response.
```python
self.assertEqual(
            list(method_response.keys()),
            ["name", "username", "profile_image_link", "rating", "details"],
            "Codechef:get_profile - keys mismatch",
```